### ISSUE TITLE
Bug with python3 on get_pretty_stats()

### ISSUE THREAD

**Issue by Mageti** [enigmampc/catalyst#153]
Dear Catalyst Maintainers,

Here is my environment:

* Operating System: Ubuntu
* Python Version: `Python 3.5.2`
* How did you install Catalyst: `pip`
* Python packages:
```
pip freeze | grep catalyst
enigma-catalyst==0.4.4
```

I run a modified version of the sample [rsi_profit_target.py](https://github.com/enigmampc/catalyst/blob/master/catalyst/examples/rsi_profit_target.py)
Everything works smoothly, but when I add `log.info('Pretty stats:\n{}'.format(get_pretty_stats(results)))` at the end of the analyze() function, I have this beautiful stacktrace:

```
Traceback (most recent call last):
  File "custom.py", line 469, in <module>
    end=pd.to_datetime('2018-1-9', utc=True),
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/utils/run_algo.py", line 527, in run_algorithm
    stats_output=stats_output
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/utils/run_algo.py", line 308, in _run
    overwrite_sim_params=False,
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/exchange/exchange_algorithm.py", line 339, in run
    data, overwrite_sim_params
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/exchange/exchange_algorithm.py", line 297, in run
    data, overwrite_sim_params
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/algorithm.py", line 730, in run
    self.analyze(daily_stats)
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/exchange/exchange_algorithm.py", line 335, in analyze
    super(ExchangeTradingAlgorithmBacktest, self).analyze(stats)
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/algorithm.py", line 485, in analyze
    self._analyze(self, perf)
  File "custom.py", line 447, in analyze
    log.info('the daily stats:\n{}'.format(get_pretty_stats(results)))
  File "/home/mageti/trading_test/venv/lib/python3.5/site-packages/catalyst/exchange/utils/stats_utils.py", line 291, in get_pretty_stats
    stats.sort(key=itemgetter('period_close'))
AttributeError: 'dict_values' object has no attribute 'sort'
```
Thanks in advance for taking a look at this one.
Mageti

**Comment by fredfortier** [comment]
Thanks for the PR, really appreciated. We're focusing more on Python 3 now.

**Comment by fredfortier** [comment]
Fixed on 0.4.6 please re-open if you still notice an issue.
